README.md

```diff
@@ -72,7 +72,7 @@ Literature
 
 Just an incomplete link collection for now:
 
-- [Paper](http://www.icg.tugraz.at/Members/steinber/scatteralloc-1) by
+- [Paper](https://doi.org/10.1109/InPar.2012.6339604) by
   Markus Steinberger, Michael Kenzel, Bernhard Kainz and Dieter Schmalstieg
 
 - 2012, May 5th: [Presentation](http://innovativeparallel.org/Presentations/inPar_kainz.pdf)
```

Comment:
@psychocoderHPC @slizzered does one of you still have this presentation?
The domain does not host it anymore and I cannot find a copy on the internet archive for it...

Otherwise we might want to contact the authors, maybe they still have it.

Reply:
sry I missed this PR, thx for pinging me. I will check if I have the old presentation somewhere.

Reply:
I do not have the presentation, I will ping @slizzered again if he still has it.